### PR TITLE
MSEDO-258: User_management_page && MSEDO-254-Edit_user_information_password

### PR DESCRIPTION
**Pull request by benjamim-ferreira**
Tasks: MSEDO-258-User_management_page and MSEDO-254-Edit_user_information_password

### CODE REVIEW

**Comment by benjamim-ferreira** [comment]
This PR is still a draft. We're looking for the missing unitary tests

File: package-lock.json [hunk 1]
```diff
@@ -4509,6 +4509,16 @@
       "integrity": "sha512-nAihlQsYGyc5Bwq6+EsubvANYGExeJKHDO3RjnvwU042fawQTQfM3Kxn7IHUXQOz4bzfwsGYYHGSvXyW4zOGLg==",
       "dev": true
     },
+    "bindings": {
```
Comment (DanielFran) [inline package-lock.json:4512]: Revert changes

File: src/main/webapp/app/account/password/password-cancel-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<form name="cancelForm" (ngSubmit)="cancel()">
+    <div class="modal-header">
+        <h5 class="modal-title mb-1" data-cy="passwordCancelDialogHeading" jhiTranslate="password.cancel.title">Deseja sair da edição de palavra-passe?</h5>
+    </div>
+
+    <div class="modal-body">
+        <p id="custom-cancel-therapeuticRegime-heading" jhiTranslate="password.cancel.question">Ao sair irá perder toda a informação inserida.</p>
```
Comment (DanielFran) [inline src/main/webapp/app/account/password/password-cancel-dialog.component.html:7]: Change id name

File: src/main/webapp/app/account/password/password-cancel-dialog.component.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    </div>
+
+    <div class="modal-footer">
+        <button id="custom-confirm-cancel-therapeuticRegime" data-cy="entityConfirmCancelButton" (click)="cancel();goBack()" type="button" class="btn btn-danger">
```
Comment (DanielFran) [inline src/main/webapp/app/account/password/password-cancel-dialog.component.html:11]: Change id name

File: src/main/webapp/app/account/settings/settings-cancel-dialog.component.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<form name="cancelForm" (ngSubmit)="cancel()">
+    <div class="modal-header">
+        <h5 class="modal-title mb-1" data-cy="passwordCancelDialogHeading" jhiTranslate="settings.cancel.title">Deseja sair da edição de perfil</h5>
+    </div>
+
+    <div class="modal-body">
+        <p id="custom-cancel-therapeuticRegime-heading" jhiTranslate="settings.cancel.question">Ao sair irá perder toda a informação inserida.</p>
```
Comment (DanielFran) [inline src/main/webapp/app/account/settings/settings-cancel-dialog.component.html:7]: Change id name

File: src/main/webapp/app/account/settings/settings-cancel-dialog.component.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    </div>
+
+    <div class="modal-footer">
+        <button id="custom-confirm-cancel-therapeuticRegime" data-cy="entityConfirmCancelButton" (click)="cancel();goBack()" type="button" class="btn btn-danger">
```
Comment (DanielFran) [inline src/main/webapp/app/account/settings/settings-cancel-dialog.component.html:11]: Change id name

**Review by DanielFran — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (vkhsunny) [reply package-lock.json:4512]: @DanielFran the current file did not listed in our pr.

Comment (DanielFran) [reply package-lock.json:4512]: @vkhsunny 
image

File: src/main/webapp/app/layouts/navbar/navbar.component.html [hunk 6]
```diff
@@ -71,13 +71,14 @@
     <ul class="navbar-nav ml-auto">
         <!-- jhipster-needle-add-element-to-menu - JHipster will add new menu it-->
         <li ngbDropdown class="nav-item dropdown pointer navbar-icon" display="dynamic" routerLinkActive="active" [routerLinkActiveOptions]="{ exact: true }">
-            <a class="nav-link" routerLink="admin/user-management" routerLinkActive="active" id="admin-menu" data-cy="adminMenu">
+            <a class="nav-link" routerLink="account/user-info" routerLinkActive="active" id="admin-menu" data-cy="userInfo">
```
Comment (DanielFran) [inline src/main/webapp/app/layouts/navbar/navbar.component.html:74]: Should be added another option, the current one is for administration management of users

**Review by DanielFran — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/layouts/navbar/navbar.component.html (hunk 6 above)
Comment (benjamim-ferreira) [reply src/main/webapp/app/layouts/navbar/navbar.component.html:74]: I inquired @sfdsilva about this issue. We came to the conclusion that the button attached is for the user-info page.
Probably in the future we should add another button (only visible to the admin's) fot the user-management page.

image

**Comment by benjamim-ferreira** [comment]
changed by  another PR

**Pull request closed without merging**